caikit_nlp/modules/token_classification/filtered_span_classification.py

```diff
@@ -136,8 +136,16 @@ def run(
         Returns:
             TokenClassificationResults
         """
+        error.type_check("<NLP82129006E>", str, text=text)
+        error.type_check("<NLP01414077E>", float, allow_none=True, threshold=threshold)
+
         if threshold is None:
             threshold = self.default_threshold
+        if text == "":
```

Comment:
is there a reason to not throw error on this? is that for compatibility with TGIS?

Reply:
NIT: Is it worth just using the native "truthiness" of `text` here (`if not text`)?